pkg/landscape/sur/group-view.hoon

```diff
@@ -1,34 +1,66 @@
 /-  *resource, *group
 ^?
 |%
++$  app  ?(%graph %groups)
++$  uid  @uvH
+::
+::  $request: State of a join request
+::
+::    .started: Time request first sent
+::    .ship: Host of group
+::    .progress: Progress of request
+::    .share-co: Automatically share contact?
```

Comment:
Gotta say I continue not being a fan of abbreviations. the `%join` action gets the full thing, this probably should too.  
(Of course, the full thing feels large and unwieldy, but we have no cute metaphor to fall back on.)

Reply:
Also might be nice to match the ordering of items between this and `%join`.

Reply:
good points!